js/Model.js

```diff
@@ -53,6 +53,7 @@ define(
 			constructor() {
 				super();
 				const bemHelper = new BemHelper('app');
+				this.isBrowserWarningShown = ko.observable(!commonUtils.isBrowserSupported);
```

Comment:
`isBrowserSupported` seems to be function:
`const isBrowserSupported = function () {`